jasypt-spring-boot-starter/src/test/java/com/ulisesbocchio/jasyptspringbootstarter/resolver/DefaultPropertyResolverTest.java

```diff
@@ -32,29 +31,30 @@ public static void main(String[] args) {
 }
 @SpringBootTest(
         properties = {
-                "spring.config.use-legacy-processing=true"
+                "spring.config.use-legacy-processing=true",
+                "server.port=9625"
         },
         classes = TestApp.class,
-        webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
-public class DefaultPropertyResolverTest {
+        webEnvironment = SpringBootTest.WebEnvironment.DEFINED_PORT
+        )
+class DefaultPropertyResolverTest {
 
     @Value("${spring.cloud.config.server.svn.password}")
     public String secret;
 
-    @LocalServerPort
-    public int port;
+    public int port = 9625;
```

Comment:
@LocalServerPort moved packages in SB3, so hard-coded port to make this compatible when using either 2 or 3 to run tests.